_includes/use-cases/use-case-transport-accept-relationship-change.md

```diff
@@ -0,0 +1,13 @@
+Accepts a relationship change with the given change Id.
```

Comment:
RelationshipChange

Reply:
Parameters, Success and Failure headlines (and descriptions) are missing.

Reply:
This is done in the other branch.